decidim-admin/lib/decidim/admin/test/manage_attachment_collections_examples.rb

```diff
@@ -18,9 +18,9 @@
       click_link translated(attachment_collection.name, locale: :en)
     end
 
-    expect(page).to have_selector("input#attachment_collection_name_en[value='#{translated(attachment_collection.name, locale: :en)}']")
-    expect(page).to have_selector("input#attachment_collection_weight[value='#{attachment_collection.weight}']")
-    expect(page).to have_selector("input#attachment_collection_description_en[value='#{translated(attachment_collection.description, locale: :en)}']")
+    expect(page).to have_selector(%(input#attachment_collection_name_en[value="#{translated(attachment_collection.name, locale: :en)}"]))
```

Comment:
Are these necessary for the test suite to pass? 

They seem cosmetic changes, and as far as I see weren't done in `develop`/`release/0.28-stable`.

Reply:
They are some needed changes. You can check my other comments to understand what is happening.